pixel/pixel.go

```diff
@@ -202,6 +204,29 @@ func (c RGB444BE) RGBA() color.RGBA {
 	return color
 }
 
+type MonochromeVertical bool
```

Comment:
I'm not entirely sure about this name, to be honest. A single pixel doesn't have a direction (it's not horizontal or vertical). And the `Image[T]` type already makes the assumption that pixels are laid out in row major order (so, vertical).
Maybe just `Monochrome` or `Gray1` ("gray 1", because there could also be something like `Gray4` for 16-level grayscale images for example).

Reply:
@aykevl what do you think?

Reply:
In the end, I decided to go with `Monochrome`